microsoft-edge/devtools-guide-chromium/about-tools.md

```diff
@@ -79,58 +81,45 @@ https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/experimental-
+The **More Tools** (**+**) menu in the **Activity Bar** and in the **Quick View** toolbar is dynamic: it omits any tab tools that are open on that toolbar.
 
 
 <!-- ====================================================================== -->
-## Panel tools vs. Drawer tools
+## Activity Bar tools vs. Quick View tools
 
 In the **Command Menu**:
 
-* _Panel tools_ are tools which open in the main toolbar by default.
+* _Panel tools_ are tools which open in the **Activity Bar** by default.
 
-* _Drawer tools_ are tools which open in the Drawer toolbar by default, at the bottom of DevTools.  Press **Esc** to show or hide the Drawer.
+* _Quick View tools_ are tools which open in the **Quick View** toolbar by default, at the bottom of DevTools.  Press **Esc** to show or hide the **Quick View**.
 
-The **Command Menu** first lists the Panel tools, and then the Drawer tools:
+The **Command Menu** first lists the Panel tools, and then the **Quick View** tools:
 
-![The Command Menu, showing the Panel tools grouped together, and then the Drawer tools](./about-tools-images/command-menu-panel-vs-drawer-tools.png)
+![The Command Menu, showing the Panel tools grouped together, and then the Quick View tools](./about-tools-images/command-menu-panel-vs-drawer-tools.png)
 
-To move a tool to the other toolbar, right-click the tool's tab and then select **Move to bottom** or **Move to top**.
+To move a tool to the other toolbar, right-click the tool's tab and then select **Move to bottom Quick View** or **Move to top Activity Bar**.
 
 To open the **Command Menu**, press **Ctrl+Shift+P** (Windows, Linux) or **Command+Shift+P** (macOS).  Or, click the **Customize and control DevTools** (![Customize icon](./about-tools-images/customize-devtools-icon-light-theme.png)) button and then select **Run command**.
 
 
 <!-- ====================================================================== -->
 ## Closing tool tabs
 
-To close a tool tab that's on a toolbar:
-
-*  Click the **x** on the tab.
-
-   The **Elements**, **Console**, and **Sources** tool tabs cannot be closed.
-
-
-To close all of the optional tabs at once:
-
-*  Right-click an optional tab on the toolbar (a tab that has an **x** button), and then select **Close all**.
-
-   Only the **Elements**, **Console**, and **Sources** remain on the main toolbar.  (The **Drawer** toolbar is unaffected.)
+To close a tool tab that's on a toolbar, right-click the tab, and then select **Remove from Activity Bar** or **Remove from Quick View**.
 
-   If you closed all tabs on the **Drawer**, only the **Console** tab remains on the **Drawer** toolbar.  (The main toolbar is unaffected.)
+The **Elements**, **Console**, and **Sources** tool are permanent tabs and cannot be closed.
```

Comment:
Console is always present in Activity Bar and Quick View.
Elements & Sources are always present on Activity Bar, and cannot be added to Quick View or moved to Quick View.